architecture/design/docdb-change-data-capture.md

```diff
@@ -4,27 +4,27 @@
 
 ### Microservice-oriented architectures
 
-There are some microservices that require a stream of changes to the data. For example, a search system powered by a service such as Elasticsearch may be used in conjunction with the database stores the transactions. The search system requires a stream of changes made to the data in YugabyteDB.
+There are some microservices that require a stream of changes to the data. For example, a search system powered by a service such as [Elasticsearch](https://www.elastic.co/elasticsearch/) may be used in conjunction with the database which stores the transactions. The search system requires a stream of changes made to the data in YugabyteDB.
 
 ### Asynchronous replication to remote systems
 
 Remote systems such as caches and analytics pipelines may subscribe to the stream of changes, transform them and consume these changes.
 
-### Two data center deployments
+### Two datacenter deployments
 
 Two datacenter deployments in YugabyteDB leverage change data capture at the core.
 
-> Note that in this design, the terms "data center", "cluster" and "universe" will be used interchangeably. We assume here that each YB universe is deployed in a single data-center.
+> Note that in this design, the terms "datacenter", "cluster" and "universe" will be used interchangeably. We assume here that each YB universe is deployed in a single datacenter.
 
 ## Setting up CDC
 
 To set up CDC we use the CDC Streams, the commands for which can be found under [yb-admin](../../docs/content/preview/admin/yb-admin.md#change-data-capture-cdc-commands).
```

Comment:
Same comment as below... these should be absolute links to the docs site. Also omit the .md, add trailing slash.